### DISCUSSION TITLE
p198のリスト18.9のエラーメッセージのtypo?

### DISCUSSION THREAD

**Discussion by youta32449999** [budougumi0617/go_todo_app#54] [Ideas]
リスト18.9内のエラーメッセージが`unexected error`となっているのですが、`unexpected error`のtypoでしょうか?
```go
t.Fatalf("unexected error: %v", err)
```
どうぞよろしくお願いいたします。

**Comment by budougumi0617** [comment]
typoだったので正誤表とコードを修正いたしました。ありがとうございます！

https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.7
